pkg/network/usm/utils/file_registry.go

```diff
@@ -164,14 +164,25 @@ func (r *FileRegistry) Register(namespacedPath string, pid uint32, activationCB,
 	}
 
 	if err := activationCB(path); err != nil {
+		// We need to call `deactivationCB` here as some uprobes could be
+		// already attached. This could be the case even when the checks below
```

Comment:
> some uprobes could be already attached
isn't it a problem in the activationCB? wondering if we should move the responsibility of the cleanup to the activation

Reply:
Perhaps, but that is a larger change that would be better left out of this bug fix?